### PR TITLE
Moved TestGrid Registry to variable name

### PR DESCRIPTION
**Pull request by chases2**
Also adds image building/pushing of entomologist to match existing utilities

/assign @fejta 
/assign @michelle192837

### CODE REVIEW

File: testgrid/cmd/entomologist/BUILD.bazel [hunk 1]
```diff
@@ -48,3 +49,9 @@ filegroup(
     tags = ["automanaged"],
     visibility = ["//visibility:public"],
 )
+
+prow_image(
+    name = "image",
+    base = "@git-base//image",
```
Comment (fejta) [inline testgrid/cmd/entomologist/BUILD.bazel]: /hold

Why do these need a git-base image?

Comment (fejta) [reply testgrid/cmd/entomologist/BUILD.bazel]: Please use the default distroless base whenever possible

Comment (chases2) [reply testgrid/cmd/entomologist/BUILD.bazel]: Noted, and done. Thank you

**Pull request merged**